com.spotify.Client.metainfo.xml

```diff
@@ -112,6 +112,9 @@
     <release date="2018-10-25" version="1.0.92.390"/>
     <release date="2018-09-15" version="1.0.89.313"/>
   </releases>
+  <branding>
+    <color type="primary">#1ed760</color>
```

Comment:
Actually need to make this black/white, and update metainfo with an icon specifically to use for marketplaces (so the icon is not always filled with black).

Reply:
Hmmm the mono logo would look stange with just the default Flathub dark theme behind it though.

Reply:
Just dropping the commit from this PR until I can figure out a better way to do this